sdk/appconfiguration/app-configuration/src/policies/throttlingRetryPolicy.ts

```diff
@@ -128,12 +97,11 @@ export class ThrottlingRetryPolicy extends BaseRequestPolicy {
           throw err;
         }
 
-        return await waitForTimeoutOrAbortOrResolve({
-          timeoutMs: delayInMs,
-          abortSignal: httpRequest.abortSignal,
-          actionFn: () => this.sendRequest(httpRequest.clone()),
-          timeoutMessage: `Unable to fulfill the request in ${delayInMs}ms when retried.`
-        });
+        await delay(delayInMs, httpRequest.abortSignal, StandardAbortMessage);
+        if (httpRequest.abortSignal?.aborted) {
```

Comment:
It's _almost_ redundant to call abortSignal here.

Reply:
Yeah, kept it just in case. Should I remove it?
My point is.. the next sendRequest call should never be invoked if the request has been aborted before, and this felt to be the way.

Reply:
No, keep it. It's correct, it just looks redundant because technically 'delay' does the same check. But, being 'async', it's possible for abortSignal.aborted to finally take affect and have it only be available afterwards. 

(kind of like double-checked locking)